.circleci/config.yml

```diff
@@ -48,7 +48,7 @@ jobs:
           name: run tests
           command: |
             . venv/bin/activate
-            cd ~/repo/src && coverage run manage.py test && coveralls
+            cd ~/repo/src && coverage run -m pytest -v && coveralls
```

Comment:
why has this been changed to `pytest`?

Reply:
So that coveralls reports accurately. See new coverage @jonathankamau .